pom.xml

```diff
@@ -32,6 +32,7 @@
     <inceptionYear>2016</inceptionYear>
 	<properties>
 		<main-class>nl.knaw.dans.easy.license.app.Command</main-class>
+		<easy.emd.version>3.0</easy.emd.version>
```

Comment:
Anyway I see a weird build order. I'd expect schema before emd before ddm before a lot else, but get:

        [INFO] EASY Datacite Library .............................. SUCCESS [ 17.709 s]
        [INFO] EASY Business Library .............................. SUCCESS [ 34.100 s]
        [INFO] Wicket Tree ........................................ SUCCESS [  2.381 s]
        [INFO] DANS Wicket Library ................................ SUCCESS [  5.609 s]
        [INFO] DEPPRECATED DANS LDAP Library ...................... SUCCESS [  1.976 s]
        [INFO] EASY LDAP Library .................................. SUCCESS [  1.230 s]
        [INFO] EASY Fedora Library ................................ SUCCESS [  1.127 s]
        [INFO] DEPPRECATED DANS Fedora Utilities .................. SUCCESS [  3.817 s]
        [INFO] DANS SOLR Library .................................. SUCCESS [  0.758 s]
        [INFO] Archaeological Pakbon (SIKB0102) Library ........... SUCCESS [  3.505 s]
        [INFO] EASY Web-UI Application ............................ SUCCESS [01:42 min]
        [INFO] Dans Dataset Metadata Library (DDM) ................ SUCCESS [ 13.909 s]
        [INFO] EASY SWORD Service ................................. SUCCESS [ 13.339 s]
        [INFO] EASY Experimental REST API ......................... SUCCESS [ 24.226 s]
        [INFO] EASY Customized PrOAI Service ...................... SUCCESS [  5.453 s]
        [INFO] EASY Business Mocking Test Library ................. SUCCESS [  4.741 s]
        [INFO] EASY Task Prototype ................................ SUCCESS [  0.155 s]
        [INFO] EASY Repo-task Framework ........................... SUCCESS [  8.291 s]
        [INFO] EASY Batch Ingest & Update (EBIU) .................. SUCCESS [  7.299 s]
        [INFO] Task: Assign Datasets To Collections ............... SUCCESS [  6.067 s]
        [INFO] Task: Dump Metadata ................................ SUCCESS [  4.432 s]
        [INFO] Task: EASY Export .................................. SUCCESS [  4.277 s]
        [INFO] Task: Add geographic coordinates ................... SUCCESS [ 15.558 s]
        [INFO] EASY Applications Master Build ..................... SUCCESS [  0.080 s]
        [INFO] EASY Ansible Role Archetype ........................ SUCCESS [  2.486 s]
        [INFO] EASY Archive Bag ................................... SUCCESS [ 16.599 s]
        [INFO] EASY Bag Store ..................................... SUCCESS [01:00 min]
        [INFO] EASY Bag Index ..................................... SUCCESS [ 48.857 s]
        [INFO] Change accessrights of files of given datasets to CC0 SUCCESS [ 15.973 s]
        [INFO] EASY Export Dataset ................................ SUCCESS [ 31.120 s]
        [INFO] EASY Fedora Commons Repo Validate Checksums ........ SUCCESS [ 14.504 s]
        [INFO] EASY Ingest ........................................ SUCCESS [ 22.822 s]
        [INFO] EASY Stage Dataset ................................. FAILURE [ 51.951 s]
        [INFO] EASY Update File-system RDB ........................ SKIPPED
        [INFO] EASY Update SOLR Index ............................. SKIPPED
        [INFO] EASY Ingest Flow ................................... SKIPPED
        [INFO] EASY Module Archetype .............................. SKIPPED
        [INFO] EASY PID Generator Service ......................... SKIPPED
        [INFO] EASY Services Documentation ........................ SKIPPED
        [INFO] EASY Schema Definitions ............................ SKIPPED
        [INFO] Task: send easy metadata to DataCite ............... SKIPPED
        [INFO] EASY Split Multi-Deposit ........................... SKIPPED
        [INFO] EASY Springfield Tools ............................. SKIPPED
        [INFO] EASY SWORD2 Deposit Service ........................ SKIPPED
        [INFO] EASY SWORD2 DANS Examples .......................... SKIPPED
        [INFO] SWORD v2 Common Server Library (forked) ............ SKIPPED
        [INFO] update dataset metadata streams .................... SKIPPED
        [INFO] EASY Master Build .................................. SKIPPED